src/make.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function make(...args) {
+  const arr = args.slice();
+
+  return function solution(...items) {
+    if (items[0] instanceof Function) { return arr.reduce((sum, current) => sum + current, 0); }
```

Comment:
this function doesn't work in case of any other sum function than incrementing
it should be arr.reduce(items[0])